domains/jonathan-russ.json

```
@@ -0,0 +1,13 @@
+{
+  "description": "Personal Website",
+  "repo": "https://github.com/JonathanXDR/Application-Website-Frontend",
+  "owner": {
+    "username": "JonathanXDR",
+    "email": "jonathan.russ@bluewin.ch",
+    "discord": "494042822320717827"
+  },
+  "record": {
+    "CNAME": "jonathan-russ.vercel.app",
+    "TXT": "google-site-verification=ZGu5jFsoZwyt_sig1wz3iNV1-8QAzMQ7qWJ1qvRZUQE"
```

You can't use any other records with CNAME
```suggestion
    "A": ["76.76.21.21"],
    "TXT": "google-site-verification=ZGu5jFsoZwyt_sig1wz3iNV1-8QAzMQ7qWJ1qvRZUQE"
```